newIDE/app/src/MainFrame/index.js

```diff
@@ -2180,6 +2183,28 @@ const MainFrame = (props: Props) => {
     return true;
   };
 
+  const canInstallPrivateAsset = async (): Promise<boolean> => {
```

Comment:
Any chance this can be useCallback'ed to avoid this changing this at every render?